src/main/telemetry/crsf.c

```diff
@@ -267,8 +267,17 @@ int16_t     Roll angle ( rad / 10000 )
 int16_t     Yaw angle ( rad / 10000 )
 */
 
-#define DECIDEGREES_TO_RADIANS10000(angle) ((int16_t)(1000.0f * (angle) * RAD))
+/// reduce angle to +/- 180 degree. Input: angle in 0.1 degree.
+static int16_t reduce_angle_range(int16_t angle_decidegree)
+{
+    while (angle_decidegree > 1800)
+        angle_decidegree -= 3600;
```

Comment:
Betaflight [coding style](https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md#non-function-statement-blocks) is to have all blocks enclosed in curly braces `{...}`. Please fix.